README.md

```diff
@@ -484,7 +484,7 @@ In order for this implementation to be functional, it is needed by the plugin de
 
 How to decide if I want to use a plugin or not? Just install / uninstall the plugin dependency from your project :smile:
 
-Finally, after the desired plugin is installed, it will also be automatically available through the `nre`. The plugin developer should be aware of this and design the interface accordingly.
+Finally, the developer can make plugin's entry points available from nre by changing the entrypoint to `[tool.poetry.plugins.nile_plugins.nre]`. Using both cli and nre scopes in nile_plugins allows it to develop powerful plugins which are easily integrated.
```

Comment:
it's confusing that in here says to change it to `nre` while the template above says "do not change this". it's also not clear how to use both. can we add an example?